src/PerformanceSensitiveAnalyzers/UnitTests/CallSiteImplicitAllocationAnalyzerTests.cs

```diff
@@ -25,7 +25,7 @@ public class MyClass
     public void Testing()
```

Comment:
💡 A new test needs to be added that uses the `net45` reference assemblies. These do not include `Array.Empty<T>()`, and the compiler may still insert an allocation.

Reply:
Could you file an issue for that please?

Reply:
I filed #3272. Keep in mind that the new test library includes reference assemblies so it's pretty easy to write a test targeting a specific version.